### ISSUE TITLE
Cmake complains about non existent CMakeLists.txt

### ISSUE THREAD

**Issue by embed** [neloe/MongoDB-Cpp#5]
The bson and tests/gtest directories are included in the main CMakeLists.txt file, but the directories themselves are empty, so attempting to run cmake fails with 

> CMake Error at CMakeLists.txt:7 (add_subdirectory):
>   The source directory
> 
>     /home/atilla/MongoDB-Cpp/bson
> 
>   does not contain a CMakeLists.txt file.
> 
> 
> CMake Error at tests/CMakeLists.txt:11 (add_subdirectory):
>   The source directory
> 
>     /home/atilla/MongoDB-Cpp/tests/gtest
> 
>   does not contain a CMakeLists.txt file.
>

**Comment by neloe** [comment]
I apologize for taking so long to respond to this; I haven't actually looked at this project in a while.

All of the directories that have no files in them are submodules.  You can be sure that you have those files by initializing the submodules:
```
git submodule update --init --recursive
```
I have an update to the README.md that will specify this, and am pushing it now.  Does this work for you?

**Comment by embed** [comment]
The submodule URLs are git@github.com/... URLs, not https URL. AFAIK those SSH based URLs are usable by only the owner of the repository. Changing the submodule links to HTTPS should fix it.

I "resolved" my issue by simply using another library.

**Comment by neloe** [comment]
Wow. I hadn't thought of the implications of making the submodules SSH (though I've run into it as problems later down the line).  When we used this at my old job, we all had SSH so it didn't even cross my radar.
I've updated the project (both this and the bson library) to use https instead of ssh submodules.  
If you don't mind me asking, what is the library you ended up using?  Since I wrote this library, it looks like they released the new Mongo driver for C++ (which might behave similarly to this one?), but at the time I couldn't find a suitable alternative.
